### PR TITLE
add movie entity

### LINKED ISSUE #13: Create a @Enitity movie class
We should add some attributes to a created Movie class like (Title, Genre, ReleaseYear, Duration, Rating) maybe?

Steps:
Create the @Entity Movie class.
Define the attributes with the correct data types.
Implement getter and setter methods for all attributes.

### PR DESCRIPTION
**Pull request by Rishad11**
Added movie entity class.
Closes #13

### CODE REVIEW

**Comment by TrungMa1** [comment]
Is this something we need to wait for Sonarcloud to check?

**Review by Lucia560 — DISMISSED** [review]

**Comment by Lucia560** [comment]
Is quite basic so I dont think so. But it would be good if the Sonar is done today, before we start with other classes and testing.

**Comment by marcusju23** [comment]
> Is quite basic so I dont think so. But it would be good if the Sonar is done today, before we start with other classes and testing.

I would like to see it as standard on our project to make tests for new methods. What do you think? 🤔

File: src/main/java/org/example/jakarta_labb/Movie.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    public double getRating() {
+        return rating;
+    }
+
+    public void setRating(double rating) {
+        this.rating = rating;
+    }
+
+    // Lägg till fler metoder ifall det behövs
```
Comment (marcusju23) [inline src/main/java/org/example/jakarta_labb/Movie.java:61]: Remove comment and redundant space.

File: src/main/java/org/example/jakarta_labb/Movie.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String title;
+    private int releaseYear;
+    private String genre;
+    private double rating;
+
+
```
Comment (marcusju23) [inline src/main/java/org/example/jakarta_labb/Movie.java:20]: Redundant space.

**Comment by Lucia560** [comment]
TDD you mean? tests before method? either way it would be great if we create test and methods in parallel, maybe we could share the workload this way? Sonar is just a good addition, you are right.

**Review by marcusju23 — CHANGES_REQUESTED** [review]
Great contribution Rishad! 👍 Before approval I think it would be great with some tests to get some code coverage. Also address some very minor changes as mentioned in the comments.

**Comment by marcusju23** [comment]
> TDD you mean? tests before method? either way it would be great if we create test and methods in parallel, maybe we could share the workload this way? Sonar is just a good addition, you are right.

I dont mind which order, I personally always do parallel. But tests should come in the pull request, in my opinion.

**Comment by TrungMa1** [comment]
> > TDD you mean? tests before method? either way it would be great if we create test and methods in parallel, maybe we could share the workload this way? Sonar is just a good addition, you are right.
> 
> I dont mind which order, I personally always do parallel. But tests should come in the pull request, in my opinion.

I agree

**Review by TrungMa1 — CHANGES_REQUESTED** [review]
Remove all the 'public' in tests

**Review by TrungMa1 — APPROVED** [review]
Well done!

**Pull request merged**